src/demo-app/a11y/dialog/dialog-neptune-iframe-a11y.html

```diff
@@ -0,0 +1,12 @@
+<h2 md-dialog-title>Neptune</h2>
```

Comment:
I suspect that having the focus start _after_ all of the dialog content is weird

@tinayuangao @crisbeto what do you think of adding a dialog config option like `focusDialogElementOnOpen` (and also `ariaLabel`) to help with dialogs like this?